### ISSUE TITLE
Toolbox

### ISSUE THREAD

**Issue by uwbwsvd** [fafalone/tbShellLib#12]
I have just imported from the internet the tbshellLib. Should I see a toolbox?
![grafik](https://user-images.githubusercontent.com/41230018/223783404-6378a8a8-7528-4872-bd09-930cc956b3ef.png)

**Comment by fafalone** [comment]
No. The Toolbox only shows ActiveX controls you can place on the form, and tbShellLib does not contain any. While some interfaces are used to create controls, e.g. IExplorerBrowser, these are done via runtime calls, not form designer editing.
